src/fedora_easyfix/main.py

```diff
@@ -21,12 +21,38 @@
 """
 
 import click
-from flask import Flask, render_template
 from fedora_easyfix.__init__ import __version__
+from fedora_easyfix.utilities.consumer import (
+    ErraticReturns,
+    TicketDataRetrieval,
+)
+from flask import Flask, jsonify, render_template, request
 
 main = Flask(__name__)
 
 
+@main.get("/endpoint/")
+def endpoint():
+    command = request.args.get("command")
+    try:
+        if command == "PREL":
+            return_data = TicketDataRetrieval().preliminary_information()
+        elif command == "REPO":
+            forge = request.args.get("forge")
+            repository = request.args.get("repository")
+            return_data = TicketDataRetrieval().repository_information(forge, repository)
```

Comment:
I didn't quite understand this.

Reply:
I mean: `dict.get(key)` will return `None` if `key` doesn't exist in the dictionary. Can the code using `forge`, `repository`, `number` deal with that sensibly? Otherwise, using e.g. `forge = request.args["forge"]` might be better because it raises the error early.

Reply:
No, it doesn't so I'd rather have an early error than to have an error mishandled. That's a good point.